spec/commands/v2/discard_draft_spec.rb

```diff
@@ -122,13 +126,16 @@
         end
 
         it "sends the published content item to the draft content store" do
-          allow(ContentStoreWorker).to receive(:perform_in)
+          expect(Presenters::ContentStorePresenter).to receive(:present).with(
+            published_item,
+            instance_of(Event),
+          )
 
-          expect(ContentStoreWorker).to receive(:perform_in)
+          allow(PresentedContentStoreWorker).to receive(:perform_async)
```

Comment:
In this case, two requests are made downstream to the content store.

1) To delete the content item at the previous base path
2) To put the content item of the live content item in place of the draft

Because of this, we need to allow the `PresentedContentStoreWorker` to receive any request in addition to the specific one we're asserting against.